data/covering_grammar/lib/make_test_file.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+"""Makes test file.
+
+Takes the gold data and the model output, and creates a three-column TSV where
+each line has a word, its gold pronunciation, and the predicted pronunciation.
+Assumes that the input files have the same words in the same order.
+"""
+
+import argparse
+import logging
+
+
+def main(args: argparse.Namespace) -> None:
+    with open(args.gold, "r") as gf, open(args.pred, "r") as pf:
```

Comment:
I don't know if this is a plus here or not but when you're doing a lot of file-opening, contextlib's `ExitStack` helps a lot: https://docs.python.org/3/library/contextlib.html#contextlib.ExitStack